src/Apps/ArtQuiz/Components/ArtQuizResultsRecommendedArtworks.tsx

```diff
@@ -47,15 +47,16 @@ export const ArtQuizResultsRecommendedArtworksFragmentContainer = createFragment
   ArtQuizResultsRecommendedArtworks,
   {
     me: graphql`
-      fragment ArtQuizResultsRecommendedArtworks_me on Me {
+      fragment ArtQuizResultsRecommendedArtworks_me on Me
+        @argumentDefinitions(limit: { type: "Int" }) {
         quiz {
           savedArtworks {
             layer(id: "main") {
-              artworksConnection {
+              artworksConnection(first: $limit) {
```

Comment:
Isn't it the case that the limit should be applied to the entire set of recommendations and not `per` each individual artwork?

Reply:
no, it's per artwork.

Reply:
Ah, yep you're right. This works for me